Clarify common questions in metadata RC (#12074)
ppy/osu-wiki: wiki/Ranking_criteria/Metadata/en.md

- **[Primary metadata sources](/wiki/Beatmap/Primary_metadata_source) must be used as reference.** You may modify metadata from primary sources only in ways allowed or required by the Ranking Criteria. If no primary sources are available, use what is most recognisable.
- **All difficulties in a beatmap set must have identical `Title`, `Artist`, `Tag`, `Source`, and `BeatmapSetID` fields.**

> - **`Artist` or `Title` fields over 81 characters must be shortened.**  <!-- Note: this rule can be dropped with no replacement if lazer allows for more flexible metadata uploads -->

Reviewer: ```suggestion
- **`Artist` or `Title` fields over 81 characters must be shortened.** <!-- this rule matches a technical limitation of BSS and can be removed if the issue is fixed -->
```
more accurate I think (at least lazer specifically has nothing to do with this)

### Allowances

> - **When multiple artists are involved in a track without clear formatting, you can use an allowance below to combine them into one artist field:**

Reviewer: ```suggestion
- **When multiple artists are involved in a track without clear formatting, you may use an allowance below to combine them into one artist field:**
```

Author: sure

## Romanisation

### Rules

> - **Romanisation rules only apply when you romanise the metadata yourself. If you use an official translation or romanisation, you must use it as-is in the romanised fields instead.**

Reviewer: this seems wrong though. this line is supposed to apply to the entire "Romanisation" section, not just "Romanisation > Rules"

if someone missed that then that's their fault imo. RC was fine here already

## Notes

[^title-case]: Capitalise all major words and leave conjunctions such as `and`, `to`, `or` as well as articles like `the`, `a`, `an` lowercase.

> [^character-voice-actor]: `Character Voice (CV)` is used when song is sung in character by the character's voice actor. `Voice Over (VO)` is used when a song is sung in character by someone other than the character's voice actor.

Reviewer: ```suggestion
[^character-voice-actor]: `CV` (*character voice*) is used when song is sung in character by the character's voice actor. `VO` (*voice over*) is used when a song is sung in character by someone other than the character's voice actor.
```
"CV" and "VO" are the things that are actually written in metadata so that should be the only literal-styled part. the other words are just for explanation

Author: ppl thought this was a spelling error and anything that makes this clear that it just isnt works so i changed it to your variant lol
